scripts/kopernikus-network.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -e
+
+if ! [[ "$0" =~ scripts/kopernikus-network.sh ]]; then
+  echo "must be run from repository root"
+  exit 1
+fi
+
+COMMAND='start'
+CAMINO_EXEC_PATH='../camino-node/build/camino-node'
+if [ $# == 1 ]; then
+    COMMAND=$1
+else
+  if [ $# == 2 ]; then
+    COMMAND=$1
+    CAMINO_EXEC_PATH=$2
+  fi
+fi
+
+echo CAMINO_EXEC_PATH: ${CAMINO_EXEC_PATH}
+CAMINO_ROOT_PATH="${CAMINO_EXEC_PATH%/*/*}"
+
+if [ "$COMMAND" = 'start' ]; then
+  echo 'Starting network runner with kopernikus configuration...'
+  ./bin/camino-network-runner server &>/dev/null &
+  curl -X POST -k localhost:8081/v1/control/start -d '{"execPath":"'${CAMINO_EXEC_PATH}'","numNodes":2,"logLevel":"INFO","globalNodeConfig":"{\"http-host\":\"0.0.0.0\",\"network-id\":\"kopernikus\",\"genesis\":\"'${CAMINO_ROOT_PATH}'/dependencies/caminoethvm/caminogo/genesis/genesis_kopernikus.json\"}"}'
```

Comment:
Is it possible to remove the requirement of the genesis file? This genesis file thing requires the code checked out what is not needed in every test environments (like caminojs).
I prefer that if network-id is given and if the name is a known network (constants in caminogo), the genesis should be 
optional.

Reply:
Hi @c4t-ag , after consulting with Alex, I have created this task https://c4t.atlassian.net/browse/TECH-593 for investigating how the runner could be adjusted to support your idea of using the --networkId flag. He is of the opinion to merge the https://github.com/chain4travel/camino-network-runner/pull/24 to unblock the e2e tests until we implement TECH-593.

Reply:
this PR will be closed . Changes will be implemented via https://github.com/chain4travel/camino-network-runner/pull/25